src/test/java/unit/DataSourceReferenced.java

```diff
@@ -1,32 +1,44 @@
 package unit;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 import bio.terra.cli.businessobject.resource.DataSource;
-import bio.terra.cli.serialization.userfacing.UFWorkspace;
 import bio.terra.cli.serialization.userfacing.resource.UFDataSource;
+import bio.terra.cli.serialization.userfacing.resource.UFGitRepo;
 import bio.terra.cli.service.WorkspaceManagerService;
 import com.fasterxml.jackson.core.type.TypeReference;
 import com.google.common.collect.MoreCollectors;
 import harness.TestCommand;
 import harness.baseclasses.SingleWorkspaceUnit;
+import harness.utils.ExternalGCSBuckets;
 import harness.utils.TestUtils;
 import harness.utils.WorkspaceUtils;
 import java.io.IOException;
 import java.util.List;
 import java.util.UUID;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.json.JSONObject;
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.api.Test;
 
 /** Tests for the `terra resource` commands that handle data sources. */
 @Tag("unit")
 public class DataSourceReferenced extends SingleWorkspaceUnit {
-  private static final String thousandGenomesWorkspaceName = "1000 Genomes";
-  private static final String thousandGenomesShortDescription = "short description";
-  private static final String thousandGenomesVersion = "version";
-  private static final String thousandGenomesResourceName =
+  private static final String THOUSAND_GENOMES_WORKSPACE_NAME = "1000 Genomes";
+  private static final String THOUSAND_GENOMES_SHORT_DESCRIPTION = "short description";
+  private static final String THOUSAND_GENOMES_VERSION = "version";
+  private static final String THOUSAND_GENOMES_RESOURCE_NAME =
```

Comment:
this test creates a workspace to use as a data source, but it doesn't seem like we're touching the actual 1000 genomes dataset at all. are we planning to in this test in the future? otherwise please update these names